### PR TITLE
Added edge case config with no-field request and solicit to ct tests

### PR DESCRIPTION
**Pull request by miklosduma**

### CODE REVIEW

File: apps/svc_sequencer/ct_test/svc_sequencer_event_SUITE_data/EdgeCases.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mix name="EdgeCases">
+	<service name="Sequencer" provision="sequencer"/>
+	<service name="Null"/>
+	<field name="OK"/>
+	<field name="START"/>
+	<solicit name="SolicitNoFieldResponse" service="Sequencer" fields="START">
+		<response name="NoFieldResponse" fields=""/>
```
Comment (lucafavatellawork) [inline apps/svc_sequencer/ct_test/svc_sequencer_event_SUITE_data/EdgeCases.xml:8]: Q1. Can a solicit have side effects of its own e.g. assuming a case that no downstream request-reply is called (as in this case)? If not, I see the definition of a solicit without response fields useful for implementing a "dummy ping" of the node.

Q2. Can the solicit be defined with a service that is not `sequencer` but rather e.g. `expr`, and with an `expr.src` nested like in [this request (not solicit) example](https://github.com/opensparkl/sse_cli/blob/caf84ab26a9e185a257098100665343c2a2c580c/docs/examples/hello-world3.xml#L34-L35), i.e. potentially with side effects?

Q3. If a consume were present in the same mix, would that consume have been called? I expect yes. E.g. the consume could keep a counter of how many times any solicit in the mix is called, or keep track of the timestamp of the most recent solicit on the mix.

Q4. (Way out of scope of this PR.) If the mix had a consume with side effects e.g. read the state of a `expr` service and toggle it (if read "true" then write "false" and if read "false" then write "true"), what would happen if two calls happened almost at the same time from two concurrent clients? Would the final result on the state be equivalent to a serialization of the two calls?

Comment (miklosduma) [reply apps/svc_sequencer/ct_test/svc_sequencer_event_SUITE_data/EdgeCases.xml:8]: As @jacobythwaites pointed it out at some point, such a solicit is basically a notify. For example, if there is a consume operation on the mix, SSE will try to satisfy that consume even though the response could be immediately sent.

Nope, solicit and notify operations MUST be on a sequencer-type service. Their client can be some other service though. For example, a notify can have an expr type service as its client, in that case the notify can change the state of the service (or be fired on a state change) if I remember correctly.

Yes :) 

That is a pretty good question. Maybe @jacobythwaites can help with that one.
